app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -127,6 +127,7 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
 
         setupInMainProcessOnly()
 
+        downloadWallpapers()
```

Comment:
I almost missed this one, this should be under a feature validation `FeatureFlags.showWallpapers` + `FeatureFlags.isThemedWallpapersFeatureEnabled`.

Reply:
Maybe we could add `FeatureFlags.isThemedWallpapersFeatureEnabled` later

Reply:
🙏 good catch, TYVM